client/my-sites/stats/stats-email-summary/index.jsx

```diff
@@ -43,87 +45,105 @@ const StatsEmailSummary = ( { translate, period, siteSlug } ) => {
 	}
 	const navigationItems = [ { label: backLabel, href: backLink }, { label: title } ];
 
+	const isStatsNavigationImprovementEnabled = config.isEnabled( 'stats/navigation-improvement' );
+
 	return (
-		<Main className="has-fixed-nav" wideLayout>
+		<Main className="has-fixed-nav" fullWidthLayout>
 			<PageViewTracker
 				path={ `/stats/${ module }/:site` }
 				title={ `Stats > ${ titlecase( module ) }` }
 			/>
-			<NavigationHeader className="stats-summary-view" navigationItems={ navigationItems } />
-			<div id="my-stats-content" className="stats-summary-view stats-summary__positioned">
-				<div className="stats-summary-nav">
-					<div className="stats-summary-nav__header">
-						<div>
-							<div className="stats-section-title">
-								<h3>{ translate( 'Stats for Emails' ) }</h3>
+			<div className="stats stats-summary-view">
+				{ isStatsNavigationImprovementEnabled && (
+					<PageHeader
```

Comment:
Not going to add download CSV here... yet...

Reply:
sounds good. download CSV is a pain if I remember correctly. Are we planning to add it in shortly though?

Reply:
Cooling down cycle I think. We'll have the rest of the week and next week as cooling down cycle 😄